docs/src/dedicated-gen-3/architecture/_index.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
:question: I'm not fully sure of the purpose of this page. The first part seems like something for the overview and the Storage and services sections don't seem like specifying the cluster itself. Do you understand the logical divide here? Or is this something more to be addressed in  #2914?

Reply:
Temporary structure in place now, it's cleaner but will be updated (will split #2914 into 1 PR = 1 page to avoid bottleneck as I'm now the only tech writer).